src/components/textarea/textarea.component.tsx

```diff
@@ -109,7 +198,9 @@ const Textarea = ({
 
   useEffect(() => {
     if (rows) {
-      minHeight.current = inputRef.current.scrollHeight;
+      /* istanbul ignore next */
+      minHeight.current =
+        inputRef && inputRef.current ? inputRef.current.scrollHeight : 0;
```

Comment:
```suggestion
       inputRef?.current?.scrollHeight || 0;
```

Reply:
if you make the change suggested by Kuba I think you might be able to remove the `istanbul ignore next`. If not, it should go just before the `0` rather than ignoring the whole line.

Reply:
yes, `istanbul ignore` would not be needed after doing that